i18n/es/OWNERS

```diff
@@ -0,0 +1,5 @@
+approvers:
+  - raelga
+
```

Comment:
Multiple approvers are preferred because the approver can't approve yourself.

Reply:
Also, dashboard maintainer can not review each languages.

Reply:
@electrocucaracha mind joining as approver?